curvefs/src/client/main.c

```diff
@@ -54,21 +54,38 @@ static const struct fuse_lowlevel_ops curve_ll_oper = {
     .statfs     = FuseOpStatFs,
 };
 
+void print_option_help(const char* o, const char* msg) {
+    printf("    -o %-20s%s\n", o, msg);
+}
+
+void extra_options_help() {
+    printf("\nExtra options:\n");
+    print_option_help("fsname", "[required] name of filesystem to be mounted");
+    print_option_help(
+        "fstype", "[required] type of filesystem to be mounted (s3/volume)");
+    print_option_help("conf", "[required] path of config file");
+    print_option_help("mdsAddr", "mdsAddr of curvefs cluster");
```

Comment:
Why pick mdsAddr out? If so, the three mds address are all needed.

Reply:
this is for multi clients situation multi backend clusters
client A -> backend cluster A
client B -> backend cluster B

the only difference between multi clients' conf is the mdsAddr parameter

so we need add the ability to set mdsAddr using cmd to avoid too many conf files, it's messy

old: client number * 1 conf file per client
new: 1 conf file shared and different mdsAddr set by cmd